optimize/self-managed/optimize-deployment/configuration/history-cleanup.md

```diff
@@ -18,12 +18,6 @@ There are four types of history cleanup:
 
 By default, all four types of history cleanup are disabled. They can be enabled individually by config and the cleanup is applied accordingly.
```

Comment:
I just realised that this line and the above list is now wrong. For C8, I think we should just include the process data and external variable cleanup

Reply:
Removing decision data and external event cleanup and adjusting this line.